plugins/ripgrep/README.md

```diff
@@ -10,4 +10,4 @@ plugins=(... ripgrep)
 
 Completion is taken from the ripgrep release [`0.10.0`](https://github.com/BurntSushi/ripgrep/releases/tag/0.10.0).
```

Comment:
This link should also be updated to point to the latest release [`11.0.2`](https://github.com/BurntSushi/ripgrep/releases/tag/11.0.2)

Reply:
Good find. I have updated it